Source/NSOutlineView.m

```diff
@@ -440,11 +457,57 @@ - (CGFloat) indentationPerLevel
  */
 - (BOOL) isExpandable: (id)item
 {
-  if (item == nil)
+  BOOL result = NO;
+  GSKeyValueBinding *theBinding = [GSKeyValueBinding getBinding: NSContentBinding
+						      forObject: self];
+  if (theBinding != nil)
     {
-      return NO;
+      BOOL leaf = YES;
+      id observedObject = [theBinding observedObject];
```

Comment:
I will need to inspect this usage of binding in more detail. At first look this is rather surprising as it breaks the loose coupling that bindings should provide but there may be valid reasons for this approach.

Reply:
The issue here was that the selection changes were being reflected in the tree controller based on what was selected in the outline view.  I was unsure of how to achieve this without doing this.

Reply:
I won't resolve this until you say okay.

Reply:
What I can say is that the behavior is the same as on macOS.   Whether it's not the same way is another matter. :/